ccmlib/node.py

```diff
@@ -556,6 +559,40 @@ def watch_log_for(self, exprs, from_mark=None, timeout=600, process=None, verbos
                         if process.returncode == 0:
                             return None
 
+    def watch_log_for_no_errors(self, exprs, from_mark=None, timeout=600, process=None, verbose=False, filename='system.log'):
+        """
+        Watch the log until one or more (regular) expression are found.
```

Comment:
expressionS :-)

Reply:
I copy/pasted from watch_log_for and didn't read =D

Reply:
I didn't check the other one. :-)